earth_enterprise/rpms/build.gradle

```diff
@@ -43,6 +47,21 @@ apply plugin: "nebula.ospackage"
 
 import org.opengee.shell.GeeCommandLine
 
+// Varables used for specifying location of two files
+// 'gee_long_version.txt' and 'gepgcommon'
+def working_dir = "../src/"
+
+// Check for either "-PRELEASE" or "-POPTIMIZE" command line parameters.
+// Return the correct directory based on which paramater is used.
+def native_x86 = {
+  if ( project.hasProperty( 'RELEASE' ) ) { return "NATIVE-REL-x86_64" }
```

Comment:
Grumble: This looks like it should be one variable with 3 possible values, rather than 3 boolean variables that are mutually exclusive.  It also seems more usual to me to have build default to release, rather than debug, which is pretty much only useful to developers.  But this mirrors the state of affairs in the `scons` build scripts, so it's OK for now.

Reply:
I like having the same interface as scons but since the default was previously "release" we should leave that the default so that we don't surprise anyone.